### PR TITLE
Issue #4970: Remove serverTTLAttribute from ldapRegistry-3.0

### LINKED ISSUE #4970: Remove serverTTLAttribute from ldapRegistry-3.0
Remove the 'serverTTLAttribute' from Liberty's metatype.xml for LDAP's attribute cache.

### PR DESCRIPTION
**Pull request by agiridharan**
Remove the 'serverTTLAttribute' from Liberty's metatype.xml for LDAP's attribute cache.

fixes #4970

### CODE REVIEW

**Comment by agiridharan** [comment]
fixes #4970

**Comment by agiridharan** [comment]
#build

File: dev/com.ibm.ws.security.wim.adapter.ldap/src/com/ibm/ws/security/wim/adapter/ldap/LdapConnection.java [hunk 1]
```diff
@@ -160,7 +159,7 @@
     private int iAttrsSizeLmit = 2000;
 
     /** default server time to live */
-    private String iServerTTLAttr = null;
+    private final String iServerTTLAttr = null;
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.adapter.ldap/src/com/ibm/ws/security/wim/adapter/ldap/LdapConnection.java]: This field should be removed, I would expect.

**Comment by agiridharan** [comment]
#build

**Comment by jvanhill** [comment]
@agiridharan - You are going to need to rebase and rerun your build. You had hundreds of failures and errors that I don't believe are related to you, but I don't feel comfortable approving this with that many failures and errors.

**Comment by agiridharan** [comment]
#build

**Pull request merged**